app/auth/cli_auth.py

```diff
@@ -37,6 +39,24 @@
 SCOPE = ["openid"]
 
 
+class RenkuCLIGitlabAuthHeaders:
+    def process(self, request, headers):
+        if not request.authorization:
+            return headers
+
+        access_token = request.authorization.password
```

Comment:
Oh, does flask automatically parse the basic authorization header into username password? Nice.